### DISCUSSION TITLE
importação de clientes

### DISCUSSION THREAD

**Discussion by diegorcampos** [RamonSilva20/mapos#1878] [Q&A]
Boa Tarde.
Há alguma maneira de importar meus clientes, que estão em uma tabela do excel para o sistema ?
Pois são mais de 700 clientes e respectivas OS e como é uma grande quantidade, fica inviável acrescentar cliente por cliente.
Tenho noção de BD, exportação e conversão de extensão, já tentei, mas o phpmyadmin, informou que outra tabela do mesmo banco havia sido corrompida.
Aguardo resposta, pois não é somente da minha loja, mas também de uma outra que apresentei o MAPOS e os mesmos gostaram.

**Comment by lukasabino** [comment]
Tem como via insert, porém tem de tratar a planilha e gerar os insers.

Importar de maneira bruta no PHPMYADMIN normalmente da pau mesmo.
